spec/lib/company_fetcher_bot_spec.rb

```diff
@@ -85,6 +85,12 @@ module UsXxCompaniesFetcher
     end
   end
 
+  describe "#statsd_namspace" do
+    it "uses the inferred_jurisdiction_code method defined in this module gives underscores" do
```

Comment:
logic is fine, but can we reword this to "uses the inferred_jurisdiction_code method defined in this module to produce jurisdiction_code in snake_case format"

Reply:
Oh yeah. That wasn't even a proper sentence. Going with your suggestion. 🐍 snake_case (done)